packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -381,6 +355,41 @@ export const useChatMessageCommonStyles = makeStyles({
       }
     }
   },
+  bodyWithPlaceholderImage: {
+    '& img[src=""]': {
+      display: 'block',
+      position: 'relative',
+      marginBottom: '5px'
+    },
+    '& img[src=""]:after': {
+      backgroundColor: tokens.colorNeutralBackground1Selected,
+      content: `url("data:image/gif;base64,R0lGODlhAQABAAAAACw=")`,
+      backgroundSize: 'center',
+      position: 'absolute',
+      width: '100%',
+      height: '100%',
+      top: '0',
+      left: '0',
+      display: 'block'
+    },
+    '& .broken-image-wrapper': {
+      width: '12rem',
```

Comment:
We hardcode this to 12rem in bodyWithPlaceholderImage as well.

Can we create a common const at the top of the file?